src/OrbitBase/File.cpp

```diff
@@ -281,4 +295,20 @@ ErrorMessageOr<std::vector<std::filesystem::path>> ListFilesInDirectory(
   return files;
 }
 
+ErrorMessageOr<std::string> GetFormatedFileDateModified(const std::filesystem::path& path,
+                                                        absl::string_view time_format) {
```

Comment:
Maybe return absl::Time here and do the formatting separately. I think we already have the formatTime function somehwere.

Reply:
Thanks for the suggestion!
For the time formatting, we have `FormatTimeWithUnderScores` defined in CaptureSerializer.cpp . It is used in the capture filename and only supports the `%Y_%m_%d_%H_%M_%S` format.

Here I would like to let the time formatting in preset view to be the same as the formatting in the capture file list in the connection window, i.e., `%m/%d/%Y %H:%M %p`.

Reply:
Still a good idea to separate formatting from getting modified time. You can add a separate function for formatting time and test it separately.

Reply:
Make sense! Changed to return `absl::Time` here. Thanks!